python/pyspark/ml/torch/distributor.py

```diff
@@ -407,13 +418,6 @@ def _run_local_training(
         try:
             if self.use_gpu:
                 gpus_owned = get_gpus_owned(self.sc)
-
```

Comment:
This is actually no longer needed since if `num_processes > len(gpus_owned)`, then we set `num_processes = len(gpus_owned)`